Server crash: `java.lang.NoClassDefFoundError: net/minecraft/client/util/ITooltipFlag`

After update `EnderStorage-continuation` to version `2.6.1` I cant start dedicated server:
```ruby
Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from EnderStorage (enderstorage)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/util/ITooltipFlag
    at java.lang.Class.getDeclaredMethods0(Native Method)
    at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
    at java.lang.Class.privateGetPublicMethods(Class.java:2902)
    at java.lang.Class.getMethods(Class.java:1615)
    at net.minecraftforge.fml.common.eventhandler.EventBus.register(EventBus.java:82)
    at codechicken.enderstorage.proxy.Proxy.preInit(Proxy.java:34)
```

[crash-2024-06-19_07.25.59-server.txt](https://github.com/user-attachments/files/15897082/crash-2024-06-19_07.25.59-server.txt)

--- a/src/main/java/codechicken/enderstorage/item/ItemEnderPouch.java
+++ b/src/main/java/codechicken/enderstorage/item/ItemEnderPouch.java
@@ -66,6 +66,8 @@ public class ItemEnderPouch extends Item implements IBakeryProvider {
         stack.getTagCompound().setBoolean(key, !currentValue);
     }
 
+
+    @SideOnly(Side.CLIENT)
     @Override
     public void addInformation(ItemStack stack, @Nullable World worldIn, List<String> tooltip, ITooltipFlag flagIn) {
         Frequency freq = Frequency.readFromStack(stack);

--- a/src/main/java/codechicken/enderstorage/recipe/RecipeWithNBT.java
+++ b/src/main/java/codechicken/enderstorage/recipe/RecipeWithNBT.java
@@ -63,7 +63,7 @@ public class RecipeWithNBT extends IForgeRegistryEntry.Impl<IRecipe> implements 
 
     @Override
     public ItemStack getRecipeOutput() {
-        return ModItems.enderPouch.getDefaultInstance();
+        return new ItemStack(ModItems.enderPouch);
     }
 }
 

